index.css

```diff
@@ -632,3 +647,92 @@ ul a {
 .shaping-img {
   margin: 23px 121px 8px 120px;
 }
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-section form {
+  border-bottom: 1px solid gray;
+}
+
+.contact-section p {
+  color: #172b4d;
+  font-family: "inter", sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  margin: 120px 24px 73px 24px;
+  text-align: center;
+}
+
+.language-container p:hover {
+  text-decoration: underline #87ceeb;
+  border: 2px solid #87ceeb;
+}
+
+.input-contact {
+  height: 18px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  margin: 33px 40px 0 40px;
+  border: none;
+}
+
+.contact-section_line1 {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contact-section textarea {
+  background-color: #fbf8f7;
+  margin: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 12px;
+  width: 81%;
+}
+
+.contact-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  padding: 12px;
+  background: #ff6b00;
+  border: none;
+  width: 40%;
+  color: #fff;
+  margin: 24px auto;
+  margin-bottom: 136px;
+}
+
+.contact-button:hover {
+  text-decoration: underline #87ceeb;
+  border: 2px solid #87ceeb;
+}
+
+.contact-section ul {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  padding: 30px 0;
+}
```

Comment:
I'm impressed by the hard work you put into this project, keep rocking 🔥. However, please let's stick to Figma designs and color. Consider using the correct color for these icons to stick to the design.

your design:
<img width="265" alt="chrome_aVXfjmMQcm" src="https://user-images.githubusercontent.com/81468675/149024963-531d3ed6-b587-4fea-ab8c-cf654897aa25.png">

Figma design:
<img width="147" alt="chrome_0qEAspUDv5" src="https://user-images.githubusercontent.com/81468675/149024988-7e36883e-7caa-4454-b537-a41afff3ff85.png">